src/renderers/webgl/WebGLShadowMap.js

```diff
@@ -297,17 +295,9 @@ function WebGLShadowMap( _renderer, _objects, maxTextureSize ) {
 
 			var useMorphing = false;
 
-			if ( material.morphTargets === true ) {
-
-				if ( geometry.isBufferGeometry === true ) {
-
-					useMorphing = geometry.morphAttributes && geometry.morphAttributes.position && geometry.morphAttributes.position.length > 0;
+			if ( material.morphTargets === true && geometry.isBufferGeometry === true ) {
```

Comment:
Do we need the ` && geometry.isBufferGeometry === true` check?

Reply:
Not anymore 😊 🎉